src/tools/illink/src/ILLink.Shared/TrimAnalysis/HandleCallAction.cs

```diff
@@ -1216,6 +1278,23 @@ private IEnumerable<MultiValue> ProcessGetMethodByName (TypeProxy type, string m
 				yield return NullValue.Instance;
 		}
 
+		private IEnumerable<MultiValue> ProcessGetMethodByNameWithPrefix (TypeProxy type, string methodName, bool nameIsPrefix, BindingFlags? bindingFlags)
+		{
+			bool foundAny = false;
+			foreach (var method in GetMethodsOnTypeHierarchyWithPrefix (type, methodName, nameIsPrefix, bindingFlags)) {
+				MarkMethod (method.RepresentedMethod);
+				yield return method;
+				foundAny = true;
+			}
+
+			// If there were no methods found the API will return null at runtime, so we should
+			// track the null as a return value as well.
+			// This also prevents warnings in such case, since if we don't set the return value it will be
+			// "unknown" and consumers may warn.
+			if (!foundAny)
+				yield return NullValue.Instance;
```

Comment:
Can a reviewer confirm if I want this section here, or not, or if I want it somewhere else? I don't know what it does (it's copied from the non WithPrefix version).